### PR TITLE
updating Expeditor configuration

### PR DESCRIPTION
**Pull request by skeshari12**
Signed-off-by: Swati Keshari <skeshari@msystechnologies.com>

The merge_actions subscription shortcut has been deprecated. All subscriptions should be declared via the subscriptions block for clarity.
Release branches are a first-class concept in Expeditor, and should be represented as such.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: .expeditor/config.yml [hunk 1]
```diff
@@ -48,40 +48,18 @@ github:
   # Which Github branches to build Omnibus releases from, and what versions
```
Comment (tduffield) [inline .expeditor/config.yml:48]: Let's move the comment as well so its associated with the new configuration.

File: .expeditor/config.yml [hunk 2]
```diff
@@ -48,40 +48,18 @@ github:
...
+      version_constraint: 16*
+  - chef-15:
+      version_constraint: 15*
+  - chef-14:
+      version_constraint: 14*
 
 changelog:
   rollup_header: Changes not yet released to stable
 
-# These actions are taken, in order they are specified, anytime a Pull Request is merged.
```
Comment (tduffield) [inline .expeditor/config.yml:62]: Let's keep this comment so its inline with the new configuration.

Comment (skeshari12) [reply .expeditor/config.yml:62]: added this comment

File: .expeditor/config.yml [hunk 3]
```diff
@@ -95,6 +72,29 @@ subscriptions:
       - built_in:publish_rubygems
       - built_in:promote_habitat_packages
       - built_in:notify_chefio_slack_channels
+  # These actions are taken, in order they are specified, anytime a Pull Request is merged.
```
Comment (tduffield) [inline .expeditor/config.yml:75]: It appears this project wants an empty space between subscriptions. Please add an empty line above the comment.

**Pull request merged**